webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric.js

```diff
@@ -118,9 +118,19 @@ export class SakaiRubric extends RubricsElement {
             :
             ""
           }
+          ${true ? html`
```

Comment:
same as above.

Reply:
Again, no need for a nested lit template, just put the content in.